code/modules/reagents/chemistry/machinery/scp_294.dm

```diff
@@ -0,0 +1,57 @@
+//////////////////////////////////////////
+//				SCP 294					//
+//										//
+//	This is a child of a chemistry		//
+//	dispenser. Info of how it works at	//
+//	http://www.scp-wiki.net/scp-294		//
+//										//
+//////////////////////////////////////////
+
+/obj/machinery/chem_dispenser/scp_294
+	name = "\improper strange coffee machine"
+	desc = "It appears to be a standard coffee vending machine, the only noticeable difference being an entry touchpad with buttons corresponding to a Galactic Common QWERTY keyboard."
+	icon = 'icons/obj/vending.dmi'
+	icon_state = "coffee"
+	amount = 10
+	resistance_flags = INDESTRUCTIBLE | FIRE_PROOF | ACID_PROOF | LAVA_PROOF
+	var/static/list/shortcuts = list(
+		"meth" = "methamphetamine",
+		"tricord" = "tricordrazine"
+	)
+	var/static/list/prohibited_reagents = list("adminordrazine")
```

Comment:
add "nanites" as well. It's a child of adminordrazine that ert get.

Reply:
done